app/controllers/describe_repair_controller.rb

```diff
@@ -1,7 +1,17 @@
 class DescribeRepairController < ApplicationController
-  def index; end
+  def index
+    @form = DescriptionForm.new
```

Comment:
Not sure if it matters, but I've been trying to keep the form and controller names in sync - so `DescribeRepairController` would reference `DescribeRepairForm`.

Reply:
Sure, that makes sense 👍